boards/raspberrypi/rpi_pico2/Kconfig.rpi_pico2

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Andrew Featherstone
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_RPI_PICO2
+	select SOC_RP2350A
+	select SOC_RP2350A_M33 if BOARD_RPI_PICO2_RP2350A_M33
+	# Breadcrumbs to ensure adding a new cpucluster (i.e. Hazard3 support)
+	# gets picked up here.
+	depends on BOARD_RPI_PICO2_RP2350A_M33
```

Comment:
```suggestion
```

Reply:
Not required, file is only included if building for this board.